### ISSUE TITLE
HEROKU-20

### ISSUE THREAD

**Issue by cantyoudobetter** [sheck/heroku-buildpack-zbar#11]
Will you be adding support for Heroku-20?

**Comment by JNajera** [comment]
Any hint on how I can, make it work for heroku-20? Thanks

**Comment by cantyoudobetter** [comment]
I need this too.  This is holding up our conversion to H-20

**Comment by sheck** [comment]
Ah sorry, for some reason I didn't see the Github notification until just now. I'll try and take a look at this sometime soon.

**Comment by sheck** [comment]
I looked into this recently and I'm having trouble getting `ruby-zbar` working locally as it seems like it's not compatible with the latest version installed by homebrew. Still planning on looking into this further.

**Comment by cantyoudobetter** [comment]
Hi sheck.  Heroku is pushing end of life now on Heroku 18 and we need to make a call on how to get this issue fixed.  Have you had a chance to look at buildpack for Heroku 20 or 22?

**Comment by sheck** [comment]
I believe the issue is with ruby-zbar as it seems like it's no longer compatible with the latest version. I will spend a bit more time today looking in to getting it working, but if I'm not able to I'm going to archive this repo as I haven't actively used it in 6-7 years now.

**Comment by sheck** [comment]
I was unable to get it to work. From what I found while working on it, it looks like there was an issue with python while trying to compile zbar. Given the following, I've decided I'm going to archive this repo:

- active development for zbar ended in 2012
- the current buildpack uses an older version that was still compatible with the ruby-zbar gem
- the ruby-zbar gem is no longer compatible with later versions of macOS
- there does not appear to be a quick fix to the python issue (I tried using `python-is-python3` but did not have success)
- I haven't personally used this in half a decade and do not have availability to continue maintaining it.

I'm bummed I wasn't able to get this working for you. Feel free to fork it and give it a shot yourself! Best of luck 👍
